### PR TITLE
Fixing comment since Actor is not used anymore.

### PR DESCRIPTION
**Pull request by danix800**
## What changes were proposed in this pull request?

Fixing comment since Actor is not used anymore.

### CODE REVIEW

**Comment by danix800** [comment]
Fixing comment since Actor is not used anymore.

**Comment by srowen** [comment]
Jenkins test this please

File: core/src/main/scala/org/apache/spark/deploy/worker/WorkerWatcher.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ import org.apache.spark.internal.Logging
 import org.apache.spark.rpc._
 
 /**
- * Actor which connects to a worker process and terminates the JVM if the connection is severed.
+ * WorkerWatcher RpcEndpoint which connects to a worker process and terminates the JVM if the connection is severed.
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/deploy/worker/WorkerWatcher.scala]: The line is too long here.

**Review by srowen — CHANGES_REQUESTED** [review]

File: core/src/main/scala/org/apache/spark/deploy/worker/WorkerWatcher.scala (hunk 1 above)
Comment (andrewor14) [inline core/src/main/scala/org/apache/spark/deploy/worker/WorkerWatcher.scala]: yeah, just say `Endpoint`

**Comment by dongjoon-hyun** [comment]
Hi, @danix800 .
Could you fix the following comment in this PR, too?

https://github.com/apache/spark/blob/master/core/src/test/scala/org/apache/spark/MapOutputTrackerSuite.scala#L256
```
// Frame size should be ~1.1MB, and MapOutputTrackerMasterActor should throw exception.
```

File: core/src/main/scala/org/apache/spark/deploy/worker/WorkerWatcher.scala (hunk 1 above)
Comment (danix800) [inline core/src/main/scala/org/apache/spark/deploy/worker/WorkerWatcher.scala]: Thanks! Fix again!

**Comment by danix800** [comment]
Hi, @dongjoon-hyun . Fixed!

**Pull request closed without merging**